### PR TITLE
TRIVIAL: Update changelog

### PR DESCRIPTION
**Pull request by ivan-nejezchleb**
Update changelog for SD-994

---

[Check PR owner responsibilities](https://confluence.intgdc.com/display/Development/Code-reviews#Code-reviews-Ownerresponsibilities)

Supported PR commands:

Command | Description
--- | ---
`extended test - examples` | Live examples tests
`extended test - storybook` | Storybook screenshot tests

See more [options](https://confluence.intgdc.com/display/kbhr/How+to+work+with+git+and+Github#HowtoworkwithgitandGithub-Extendedchecks).

# PR Checklist

- [ ] Verify code changes ([Checklist](https://confluence.intgdc.com/display/Development/Code-reviews+checklist), [Best practices](https://confluence.intgdc.com/display/Development/Code-reviews+best+practices))
- [ ] [Verify pull-request formalities](https://confluence.intgdc.com/display/Development/Code-reviews)
- [ ] Change was tested by using [gdc-dev-release](https://confluence.intgdc.com/display/~tomas.vojtasek/Private+NPM) in [gdc-analytical-designer](https://github.com/gooddata/gdc-analytical-designer) and [gdc-dashboards](https://github.com/gooddata/gdc-dashboards) (if applicable)
- [ ] Migration guide (for major changes) is mentioned in [CHANGELOG.md](../blob/master/CHANGELOG.md).
- [ ] Successful `extended test - examples`
- [ ] Successful `extended test - storybook`
- [ ] Checked yarn.lock consistency (no dep. duplicities especially Goodstrap)

# Related PRs

- gdc-analytical-designer:
- gdc-dashboards:

# Related Jira tasks

### CODE REVIEW

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- gooddata-react-components-storybook-zuul-docker : SKIPPED
- gooddata-react-components-fossa-scanning-zuul-el7 : SKIPPED (non-voting)
- gooddata-react-components-unit-tests-zuul-docker-el7 : SKIPPED
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/1966/ : FAILURE in 16s
- gooddata-react-components-examples-zuul-docker : SKIPPED

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/4269/ : SUCCESS in 12m 19s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-fossa-scanning-zuul-el7/188/ : FAILURE in 1m 59s (non-voting)
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/8145/ : SUCCESS in 9m 05s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/1967/ : SUCCESS in 16s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-examples-zuul-docker/2995/ : SUCCESS in 20m 22s

File: CHANGELOG.md [hunk 1]
```diff
@@ -21,7 +21,8 @@ The REST API versions in the table are just for your information as the values a
 
 ### Added
 - Secondary Y axis is now rotated the same way as primary Y axis in line, column, combo and combo2 charts
-- Manual resizing of Pivot table, resize all measurers columns in the table at once
+- Manual resizing of Pivot table columns, resize all measurers columns in the table at once
+- Increase pin limit in GeoChart (from 2k) to 25k
```
Comment (evgeniabogranskaya) [inline CHANGELOG.md:25]: - The secondary Y axis is now rotated the same way as the primary Y axis in line, column, and combo charts.
- Manual resizing for columns in pivot tables; resizing all measure columns in a pivot table at once
- The max number of pins in geo pushpin charts increased from 2,000 to 25,000

**Pull request merged**